app.ts

```diff
@@ -120,6 +119,9 @@ export type CompilerExplorerOptions = Partial<{
     noLocal: boolean;
 }>;
 
+const router = express.Router();
```

Comment:
so the router is created at https://github.com/compiler-explorer/compiler-explorer/blob/main/app.ts#L527
can you use that one instead of creating another?